CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 1.4.2 -- 2019-05-30
```

Comment:
Because we are adding a new dependency and doing several upgrades, I think this should probably be a minor version bump, not a patch.

https://github.com/aws/aws-encryption-sdk-java/blob/master/VERSIONING.rst

Reply:
Excellent point, thank you. The new revision fixes it.